commerce_coordinator/apps/stripe/tests/test_clients.py

```diff
@@ -3,11 +3,14 @@
 from unittest.mock import MagicMock, patch
 
 import ddt
+import mock
 import stripe
 from django.conf import settings
 from django.test import override_settings
 from rest_framework.exceptions import ValidationError
 
+from commerce_coordinator.apps.commercetools.tests.conftest import gen_order
```

Comment:
In general, in Coordinator, we try to avoid importing from outside the Django app (in this case, `stripe`), unless it's to the `core` Django app.

Is there a specific reason we need to have a Commercetools order here? If not, building a small object with an order and an amount may reduce complexity/dependencies here.